homeassistant/components/openai_conversation/config_flow.py

```diff
@@ -118,45 +115,67 @@ async def async_step_init(
     ) -> ConfigFlowResult:
         """Manage the options."""
         if user_input is not None:
-            return self.async_create_entry(title="OpenAI Conversation", data=user_input)
-        schema = openai_config_option_schema(self.config_entry.options)
+            if user_input[CONF_LLM_HASS_API] == "none":
```

Comment:
Should we make `"none"` a constant in the LLM helper?

Reply:
Since it's not a value we ever pass to the LLM API, I am not sure? We wouldn't want to just have a stub API returning no tools, as that is a waste of compute.